packages/flutter_tools/fuchsia_attach.dart

```diff
@@ -0,0 +1,128 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:args/args.dart';
+
+import 'package:flutter_tools/runner.dart' as runner;
+import 'package:flutter_tools/src/artifacts.dart';
+import 'package:flutter_tools/src/base/common.dart';
+import 'package:flutter_tools/src/base/context.dart';
+import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:flutter_tools/src/cache.dart';
+import 'package:flutter_tools/src/commands/attach.dart';
+import 'package:flutter_tools/src/commands/doctor.dart';
+import 'package:flutter_tools/src/fuchsia/fuchsia_sdk.dart';
+import 'package:flutter_tools/src/run_hot.dart';
+import 'package:flutter_tools/src/runner/flutter_command.dart';
+
+final ArgParser parser = ArgParser()
+  ..addOption('build-dir', help: 'The fuchsia build directory')
+  ..addOption('dart-sdk', help: 'The prebuilt dart SDK')
+  ..addOption('target', help: 'The GN target to attach to')
+  ..addFlag('verbose', negatable: true);
+
+// Track the original working directory so that the tool can find the
+// flutter repo in third_party.
+String originalWorkingDirectory;
+
+Future<void> main(List<String> args) async {
```

Comment:
This can be just `void` now

Reply:
We have the avoid_void_async lint enabled.